test/test_sendbuf.c

```diff
@@ -87,7 +87,7 @@ int setup_default(void **state) {
     trxmode = CWMODE;
     cwkeyer = 1;
     digikeyer = 1;
-    strcpy(my.call, "dl1jbe\n"); 		// !!! do not forget trailing \n
+    strcpy(my.call, "dl1jbe");
```

Comment:
Don't we want to test here the call sign with `/`?

Reply:
Pushed the updated test of getctydata(). The original issue (the seconds one) can be reproduced by adding a trailing NL to the calls on lines 175 and 178 of `test_getctydata.c`. The test will fail on line 178, i.e. "LA3BB\n" leads to the same (and correct) value as for "LA3BB/QRP", but "LA3BB/P\n" delivers zero.
The correction was done by dropping the legacy trailing NL, not by changing the function.